migrations/default/2_create_triggers/up.sql

```diff
@@ -0,0 +1,380 @@
+--
+-- handle crash record insert by copying
+-- to the vz_ and unified tables
+--
+create or replace function db.crashes_cris_insert_rows()
+returns trigger
+language plpgsql
+as
+$$
+BEGIN
+    -- insert new unified record
+    INSERT INTO db.crashes (crash_id, road_type_id) values (
+        new.crash_id, new.road_type_id
+    );
+    -- insert new (editable) vz record (only record ID)
+    INSERT INTO db.crashes_vz (crash_id) values (new.crash_id);
+    RETURN NULL;
+END;
+$$;
+
+create or replace trigger insert_new_crashes_cris
+after insert on db.crashes_cris
+for each row
+execute procedure db.crashes_cris_insert_rows();
+
+--
+-- handle a unit record insert by copying
+-- to the vz_ and unified tables
+--
+create or replace function db.units_cris_insert_rows()
+returns trigger
+language plpgsql
+as
+$$
+BEGIN
+    -- insert new unified record
+    INSERT INTO db.units (id, crash_id, unit_nbr, unit_type_id) values (
+        new.id, new.crash_id, new.unit_nbr, new.unit_type_id
+    );
+    -- insert new (editable) vz record (only record ID)
+    INSERT INTO db.units_vz (id) values (new.id);
+    RETURN NULL;
+END;
+$$;
+
+create or replace trigger insert_new_units_cris
+after insert on db.units_cris
+for each row
+execute procedure db.units_cris_insert_rows();
+
+--
+-- handle a people record insert by setting
+-- the  unit_id column based on the crash_id and unit_nbr
+--
+create or replace function db.people_cris_set_unit_id()
+returns trigger
+language plpgsql
+as $$
+DECLARE
+   unit_record record;
+BEGIN
+    SELECT INTO unit_record *
+        FROM db.units_cris where crash_id = new.crash_id and unit_nbr = new.unit_nbr;
+    new.unit_id = unit_record.id;
+    RETURN new;
+END;
+$$;
```

Comment:
like i mentioned above, we could do away with this trigger if we want to handle this within the cris import ETL. i think i prefer this option. it's simple and performant and will keep our import code clean.